aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/AeroGearGCMPushJsonConfiguration.java

```diff
@@ -0,0 +1,290 @@
+     */
+    public String getOsVersion() {
+        return pushConfig.getOsVersion();
+    }
+
+    /**
+     * The Alias is an identifier of the user of the system.
+     * 
+     * Examples are an email address or a username
+     * 
+     * @return alias
+     * 
+     */
+    public String getAlias() {
+        return pushConfig.getAlias();
+    }
+
+    /**
+     * The Alias is an identifier of the user of the system.
+     * 
+     * Examples are an email address or a username
+     * 
+     * @param alias the new alias
+     * @return the current configuration
+     * 
+     */
+    public AeroGearGCMPushJsonConfiguration setAlias(String alias) {
+        this.pushConfig.setAlias(alias);
+        return this;
+    }
+
+    /**
+     * The categories specifies a channel which may be used to send messages
+     * 
+     * @return the current categories
+     * 
+     */
+    public List<String> getCategories() {
+        return pushConfig.getCategories();
+    }
+
+    /**
+     * The categories specifies a channel which may be used to send messages
+     * 
+     * @param categories the new categories
+     * @return the current configuration
+     * 
+     */
+    public AeroGearGCMPushJsonConfiguration setCategories(List<String> categories) {
+        this.pushConfig.addCategories(categories);
+        return this;
+    }
+
+    /**
+     * The categories specifies a channel which may be used to send messages
+     * 
+     * @param categories the new categories
+     * @return the current configuration
+     * 
+     */
+    public AeroGearGCMPushJsonConfiguration setCategories(String... categories) {
+        this.pushConfig.setCategories(categories);
+        return this;
+    }
+
+    /**
+     * The categories specifies a channel which may be used to send messages
+     * 
+     * @param category a new category to be added to the current list.
+     * @return the current configuration
+     * 
+     */
+    public AeroGearGCMPushJsonConfiguration addCategory(String category) {
+        this.pushConfig.addCategory(category);
+        return this;
+    }
+
+    public AeroGearGCMPushJsonConfiguration loadConfigJson(Context context) {
+        try {
+            InputStream fileStream = context.getResources().getAssets().open(getFileName());
+            int size = fileStream.available();
+            byte[] buffer = new byte[size];
+            fileStream.read(buffer);
+            fileStream.close();
+            String json = new String(buffer);
+
+            JSONObject pushConfig = new JSONObject(json);
+            JSONObject pushAndroidConfig = pushConfig.getJSONObject("android");
+
+            this.pushConfig.setPushServerURI(new URI(pushConfig.getString("pushServerURL")));
+            this.pushConfig.addSenderId(pushAndroidConfig.getString("senderID"));
+            this.pushConfig.setVariantID(pushAndroidConfig.getString("variantID"));
+            this.pushConfig.setSecret(pushAndroidConfig.getString("variantSecret"));
```

Comment:
Not sure, I can't see any value to move it for a constants because we are using it only here. Anyway no problem to me.